operations/mimir/rollout-operator.libsonnet

```diff
@@ -37,8 +37,32 @@
                               $._config.cortex_compactor_concurrent_rollout_enabled ||
                               $._config.ingest_storage_ingester_autoscaling_enabled ||
                               $._config.enable_rollout_operator_webhook,
+
+    // do not deploy the pod eviction webhook if there is no zone pdb configured.
+    // the eviction webhook can be deployed even if there are no zone pdb configurations.
+    // pod evictions not managed by a zpdb will be allowed and contine to be handled by any other pdb configurations.
+    disable_rollout_operator_eviction_webhook_when_no_zpdb: false,
```

Comment:
I would simplify this a bit: I would have an option like `enabled_rollout_operator_eviction_webhook` that defaults to the value of `$._config.enable_rollout_operator_webhook`.